packages/@react-aria/gridlist/intl/bg-BG.json

```diff
@@ -0,0 +1,4 @@
+{
+  "hasActionAnnouncement": "row has action",
+  "hasLinkAnnouncement": "row has link: {link}"
+}
```

Comment:
I know we discussed just add the english intl files for tree's expand/collapse strings, but are we fine doing that here for gridlist's action/link announcements (concerned since this is a full released package vs tree being alpha still)?

Reply:
Decision is we will leave the translations in here for now so we can get them translated by the globalization team and remove the row announcements for now. This means gridlist row's announcements still have parity with what is on prod right now so should be fine